plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudgetingDefaults.hs

```diff
@@ -19,3 +26,9 @@ defaultCostModelParams = extractModelParams defaultCostModel
 -- import           Data.Default
 -- defaultCostModel :: CostModel
 -- defaultCostModel = def
+
+-- | Default costs for CEK machine instructions.
+defaultCekCosts :: CekCosts
+defaultCekCosts =
+  $$(readJSONFromFile "cost-model/data/cekCosts.json")
+
```

Comment:
I think that `CekCosts` is sufficiently simple that we don't need the analogue of `defaultCostModelParams` for ledger interoperation (at least not yet).

Reply:
Um. well, we probably do need them to be part of the params we expose to the ledger so they can be adjusted, right? So we'll need something like that. Maybe not now.

Reply:
> Um. well, we probably do need them to be part of the params we expose to the ledger so they can be adjusted, right? So we'll need something like that. Maybe not now.

I'm not quite sure what's required there, so let's think and come back to that. There's already an issue for this in SCP-2176.